spectractor/simulation/adr/baseobject.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python

```

Comment:
it would be great to get rid off this BaseObject class

Reply:
Still in progress